src/System.Net.Http/tests/FunctionalTests/Configurations.props

```diff
@@ -6,6 +6,7 @@
       netcoreapp-Unix;
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
```

Comment:
I don't think it is necessary to include these two files though, only PlatformDetection. These guys can be reverted.